main/src/modules/Jvm.scala

```diff
@@ -373,7 +373,7 @@ object Jvm {
                               shellClassPath: Agg[String],
                               cmdClassPath: Agg[String],
                               jvmArgs: Seq[String],
-                              shebang: Boolean = false) = {
+                              shebang: Boolean = true) = {
```

Comment:
This is a default argument of a public method. Changing the default means potentially breaking all users. This is definitely the wrong position to fix the linked issue (#834). Instead, you should provide the value on the call site.